widget/gridwrap.go

```diff
@@ -82,7 +83,12 @@ func (l *GridWrap) CreateRenderer() fyne.WidgetRenderer {
 	l.ExtendBaseWidget(l)
 
 	if f := l.CreateItem; f != nil && l.itemMin.IsZero() {
-		l.itemMin = f().MinSize()
+		item := f()
```

Comment:
Could this logic (create new item, then maybe refresh depending on theme) be extracted into a helper function? it appears 3 times. Also I'm a little concerned about calling new internal functions in widgets because does that suggest this would not be possible if writing something like the GridWrap as a custom widget?

Reply:
This is not the first internal call within GridWrap, though I appreciate the point.
I think that this new API will go away when we fix the issue that containers do not currently track when the gain or lose child items.
There are two possible approaches that would mean this becomes unneeded but both will take a lot of internal refactoring and this seemed like a good approach for now.

Reply:
OK so this inspired me to write up a short test case and it seems that yes, containers do not properly apply the theme override to their contents that are added after the fact. I think this is another blocker on theme override being complete, do you not see it that way?

Reply:
It works well enough in the Fysion use-case which is the only metric I have at this time.
When you override a container and later add items to it they would be overridden as well. It doesn't seem like a big deal to me but others may disagree.